pkg/termios/sgtty_openbsd.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021 the u-root Authors. All rights reserved
```

Comment:
I kept this copyright date as this is a copy of the FreeBSD file, which seemed to be the convention.

(All these x/sys/unix definitions are slightly different enough between GOOSes that it's annoying to try to unify them too much.)

Reply:
honestly our convention is... kind of whatever.